### PR TITLE
Posts

### PR DESCRIPTION
**Pull request by tanveertkd**
Added Follow/unfollow users
Added post operations.  

- Create post
- Delete Post
- Edit Post
- Like post
- Dislike post
- Comment
- Bookmark post

### CODE REVIEW

File: src/components/EditPostModal/EditPostModal.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const EditPostModal = ({ postData, setEditModal, token }) => {
+    const dispatch = useDispatch();
+    const [postContent, setPostContent] = useState({
+        content: postData?.content,
+    });
+
+    const handleEditSubmission = (e, postContent) => {
+        e.preventDefault();
+        dispatch(editPostHelper({ postId: postData?._id, postContent, token }));
+    };
```
Comment (himadri2110) [inline src/components/EditPostModal/EditPostModal.jsx:14]: Close the edit modal on Submission

**Review by himadri2110 — APPROVED** [review]
The code looks clean, all the functionalities work fine.
I hope you'll fix the UI bugs.
Good to go.

File: src/components/EditPostModal/EditPostModal.jsx (hunk 1 above)
Comment (tanveertkd) [reply src/components/EditPostModal/EditPostModal.jsx:14]: Fixed now. Thanks for pointing it out.

**Comment by poornima-alugubelly** [comment]
Add redirects file to app , as on refreshing it is giving 404
Posts are not really showing up on adding

**Review by poornima-alugubelly — APPROVED** [review]

**Pull request merged**